### ISSUE TITLE
"This content is blocked" error for embedded media

### ISSUE THREAD

**Issue by tuoggy** [nextcloud/cms_pico#202]
TLDR - Does Pico/Nextcloud block displaying external media, and is there any way to get it to work?

It's possible that I'm a noob and have made a dumb mistake somewhere.
I've searched Google, GitHub, and Nextcloud support to find an answer, and I can't seem to find anything.

I'm using Nextcloud 23.0.4 on a personal home server (testing for a production instance hosted on an AWS account).
I've just installed PicoCMS from the store, and added the [PicoVideoEmbed](https://git.kompot.si/kompot/PicoVideoEmbed) plugin from the wiki so that I can embed YouTube videos.
I followed the directions to add the plugin to the appdata* folder located in the data directory. Everything is owned by www-data.

The goal is to make a basic training site within the Nextcloud domain with YouTube videos and links to files within Nextcloud.

Everything works great until I go to open the a page with a video. I get a sad file image with the message, "This content is blocked. Contact the site owner to fix the issue."

I understand that PicoCMS for Nextcloud errs on the side of caution, but I was curious if this was something that I don't have setup properly, or if it just won't work due to limitations?

**Comment by PhrozenByte** [comment]
I'm afraid embedding external videos is not possible for security reasons and blocked by Pico's Content Security Policy. This isn't going to be possible until we can host websites under different domains.

**Comment by tuoggy** [comment]
Thanks for the quick response.
I was hoping this wasn't the answer, but it's good to know.
